Documentation/CRDs/Block-Storage/ceph-block-pool-rados-namespace-crd.md

```diff
@@ -81,3 +81,42 @@ parameters:
   pool: replicapool
   ...
 ```
+
+### Mirroring
+
+#### Prerequisites
+
+Enable the mirroring in the default rados namespace(CephBlockPool), refer [CephBlockPool documentation](ceph-block-pool-crd.md#mirroring).
+
+Configure the rados namespace CRD with mirroring spec
+
+```yaml
+apiVersion: ceph.rook.io/v1
+kind: CephBlockPoolRadosNamespace
+metadata:
+  name: namespace-a
+  namespace: rook-ceph # namespace:cluster
+spec:
+  # The name of the CephBlockPool CR where the namespace is created.
+  blockPoolName: replicapool
+  mirroring:
+    enabled: false
+    mode: image
+    remoteNamespace: "test"
+    snapshotSchedules:
+```
+
+#### Mirroring Spec
```

Comment:
All of the spec settings should be included in a single doc section. So let's merge these settings (lines 111-117) with the spec section starting on line 49.